app/instances.go

```diff
@@ -73,42 +73,42 @@ func (a *App) createTriggers(tConfigs []*trigger.Config, runner action.Runner) (
 		}
 
 		if tConfig.Ref == "" && tConfig.Type != "" {
-			log.RootLogger().Warnf("trigger configuration 'type' deprecated, use 'ref' in the future")
+			log.RootLogger().Warnf("trigger [%s]'s configuration 'type' deprecated, use 'ref' in the future", tConfig.Id)
 			tConfig.Ref = "#" + tConfig.Type
 		}
 
 		ref := tConfig.Ref
 
 		if tConfig.Ref == "" {
-			return nil, fmt.Errorf("ref not specified for trigger: %s", tConfig.Id)
+			return nil, fmt.Errorf("trigger [%s]'s ref not specified", tConfig.Id)
 		}
 
 		if tConfig.Ref[0] == '#' {
 			var ok bool
 			ref, ok = support.GetAliasRef("trigger", tConfig.Ref)
 			if !ok {
-				return nil, fmt.Errorf("trigger '%s' not imported", tConfig.Ref)
+				return nil, fmt.Errorf("trigger [%s] with ref [%s] not imported", tConfig.Id, tConfig.Ref)
```

Comment:
return nil, fmt.Errorf("trigger [%s]'s 'ref' [%s] not imported", tConfig.Id, tConfig.Ref)